src/main/scala/org/broadinstitute/dsde/firecloud/webservice/UserApiService.scala

```diff
@@ -129,12 +129,25 @@ trait UserApiService extends HttpService with PerRequestCreator with FireCloudRe
           passthrough(UserApiService.billingAccountsUrl, HttpMethods.GET)
         }
       } ~
-      path("profile" / "trial") {
-        post {
-          requireUserInfo() { userInfo => requestContext =>
-            perRequest(requestContext, TrialService.props(trialServiceConstructor),
-              TrialService.EnrollUser(userInfo)
-            )
+      pathPrefix("profile" / "trial") {
+        pathEnd {
+          post {
+            requireUserInfo() { userInfo =>
+              requestContext =>
```

Comment:
nit: requestContext can be moved to line above (similar to usage in other files)